.gitmodules

```diff
@@ -111,10 +111,14 @@
     path = third_party/foxi
     url = https://github.com/houseroad/foxi.git
 [submodule "third_party/tbb"]
-	path = third_party/tbb
-	url = https://github.com/01org/tbb
-	branch = tbb_2018
+    path = third_party/tbb
+    url = https://github.com/01org/tbb
+    branch = tbb_2018
 [submodule "android/libs/fbjni"]
     ignore = dirty
     path = android/libs/fbjni
     url = https://github.com/facebookincubator/fbjni.git
+[submodule "third_party/XNNPACK"]
+    path = third_party/XNNPACK
+    url = https://github.com/AshkanAliabadi/XNNPACK.git
+    branch = xnnpack_pytorch_merge_temp
```

Comment:
Intentionally point to a temporary copy of the repo that contains some modifications that we do not want to upstream.  This allows us to limit this diff's scope by taking the threadpool changes out of this diff.  I am not sure what that means for internal BUCK integration though.

Reply:
Well, regardless of what open source is pointed at, you're going to have to use whatever XNNPACK copy is living in fbcode. It's pretty confusing for developers if open source and fbcode XNNPACK versions disagree. If there is no XNNPACK copy in fbcode (or you can pin it to your fork) all is well.

Reply:
We currently don't have a copy of XNNPACK in fbcode.  The tradeoff I need to make is to either make this PR shorter and swap out the XNNPACK copy later on, or make this PR longer to include the pthreadpool changes.

Do you foresee we running into any issues given that we don't have a copy of XNNPACK fbcode at this point, and we want to swap this later to point to upstream XNNPACK?  I think at that point we can copy upstream XNNPACK into fbcode as well?

Reply:
If there is no copy of XNNPACK in fbcode at the moment, you are gucci. This tech debt is set to explode when XNNPACK moves into fbcode.